cql3/expr/expression.cc

```diff
@@ -689,12 +715,12 @@ value_set possible_lhs_values(const column_definition* cdef, const expression& e
                                 return unbounded_value_set;
                             }
                             if (is_compare(oper.op)) {
-                                const auto val = to_bytes_opt(oper.rhs->bind_and_get(options));
+                                managed_bytes_opt val = to_managed_bytes_opt(oper.rhs->bind_and_get(options));
```

Comment:
Nitpick: why change `const auto`?  That change isn't strictly necessary, methinks.

Reply:
Removing the `const` is needed to move `val` (into `to_range`). As for replacing `auto` with an explicit type, no particular reason.